packages/manager/modules/pci/src/projects/project/data-processing/submit-job/spark-sizing/spark-sizing.controller.js

```diff
@@ -59,6 +65,71 @@ export default class {
     this.state.advancedSizing = !this.state.advancedSizing;
   }
 
+  /**
+   * Handler for the checkbox allowing the editing of the driver memory overhead
+   */
+  onChangeAdvancedSizingDriverMemOverheadHandler(newValue) {
+    if (this.state.advancedSizing && newValue) {
+      // this.onChangeDriverMemoryHandler(this.state.driverMemoryGb);
+      this.updateMemoryOverHead(this.state.driverMemoryGb, newValue, 'driver');
```

Comment:
Would it make sense to 
```suggestion
      return this.updateMemoryOverHead(this.state.driverMemoryGb, newValue, 'driver');
```

Reply:
Hi Marie,
I do not believe it would make sense, no, as the updateMemoryOverhead only update some controller values and do not return anything